include/tileset_anims.h

```diff
@@ -0,0 +1,9 @@
+#ifndef GUARD_TILESET_ANIMS_H
+#define GUARD_TILESET_ANIMS_H
+
+void InitTilesetAnimations(void);
+void InitSecondaryTilesetAnimation(void);
+void UpdateTilesetAnimations(void);
+void TransferTilesetAnimsBuffer(void);
+
+#endif // GUARD_TILESET_ANIMS_H
```

Comment:
Again, text file should end with new line.

Reply:
Resolved.